stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -69,15 +69,28 @@
    */
   public abstract void prepare() throws Exception;
 
+  /**
+   * Perform post-run cleanups.
+   */
+  public void cleanup() throws Exception {}
```

Comment:
Added to the interface as some tests will include a cleanup phase like removing test files.